mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java

```diff
@@ -136,14 +134,6 @@ public void showToast(String message) {
         Toaster.showToast(getContext(), message);
     }
 
-    @Override
-    public void gotoHome() {
-        Intent intent = new Intent(getActivity(), HomeActivity.class);
-        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP |
-                Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
-        startActivity(intent);
-    }
-
```

Comment:
I think removing the call to this function is enough. This function could be used further when more features are implemented